actionview/lib/action_view/template/error.rb

```diff
@@ -49,7 +49,7 @@ def initialize(paths, path, prefixes, partial, details, *)
       searched_paths = @prefixes.map { |prefix| [prefix, path].join("/") }
 
       out  = "Missing #{template_type} #{searched_paths.join(", ")} with #{details.inspect}.\n\nSearched in:\n"
-      out += paths.compact.map { |p| "  * #{p.to_s.inspect}\n" }.join
+      out += paths.filter_map { |p| "  * #{p.to_s.inspect}\n" }.join
```

Comment:
Can you revert all `compact.map` to `filter_map` conversion? These are not same.

Reply:
Okay, I will revert them. Thanks